### PR TITLE
gcc: mention -c compiles into object file

### PR DESCRIPTION
**Pull request by jxu**

### CODE REVIEW

File: pages/common/gcc.md [hunk 1]
```diff
@@ -19,6 +19,6 @@
 
 `gcc -S {{path/to/source.c}}`
 
-- Compile source code without linking:
+- Compile source code without linking, into object file:
```
Comment (navarroaxel) [inline pages/common/gcc.md:22]: Perhaps this better
```suggestion
- Compile source code into an object file without linking:
```

Comment (jxu) [reply pages/common/gcc.md:22]: I considered that wording as well. As long as it's clear an object file by definition is without linking (I believe)

Comment (CleanMachine1) [reply pages/common/gcc.md:22]: Feel free to commit the change @jxu if it keeps the same meaning

**Review by CleanMachine1 — APPROVED** [review]
lgtm after the navarroaxel's comment is resolved.

File: pages/common/gcc.md (hunk 1 above)
Comment (sbrl) [reply pages/common/gcc.md:22]: @navarroaxel's wording reads better for me

**Comment by jxu** [comment]
I don't know if standard practice is to squash commits together, but since the first commit already got comments I'll leave it

File: pages/common/gcc.md [hunk 2]
```diff
@@ -19,6 +19,6 @@
 
 `gcc -S {{path/to/source.c}}`
 
-- Compile source code without linking:
+- Compile source code into an object file without linking:
```
Comment (EmilySeville7cfg) [inline pages/common/gcc.md:22]:
```suggestion
- Compile the source code into an object file without linking:
```

Maybe it's better. Anyway everything looks nice. 👍🏻

**Review by EmilySeville7cfg — APPROVED** [review]

File: pages/common/gcc.md (hunk 2 above)
Comment (jxu) [reply pages/common/gcc.md:22]: I don't think it makes a difference. If using "a specific" it would have to be with "source code file".

**Review by EmilySeville7cfg — APPROVED** [review]

**Comment by EmilySeville7cfg** [comment]
Merged, thanks! ❤️

**Pull request merged**